### PR TITLE
Add Kubernetes deployment

### PR DESCRIPTION
**Pull request by autrilla**
This adds a simple Kubernetes configuration that runs postgres, the api and the scanner I used a few days ago to try to reproduce a bug.

 I don't know if we prefer docker-compose (#171) over kubernetes.

### CODE REVIEW

**Comment by jvehent** [comment]
That's cool, I guess. Though I'd prefer to put that in the doc than on its own set of files that will likely become out of date in a few months, since we don't actively use it.

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ RUN go install github.com/mozilla/tls-observatory/tlsobs-api && \
     go install github.com/mozilla/tls-observatory/tlsobs-scanner && \
     go install github.com/mozilla/tls-observatory/tlsobs-runner && \
     apt-get update -y && \
-    apt-get install git libcurl4-nss-dev libnss3 libnss3-dev clang -y && \
+    apt-get install git libcurl4-nss-dev libnss3 libnss3-dev clang postgresql-client -y && \
```
Comment (jvehent) [inline Dockerfile]: Why?

Comment (autrilla) [inline Dockerfile]: Needed a way to automate creation of the database and schema from the container. fwiw this doesn't increase the image size substantially. `docker images` shows a size of 1.34GB before this, and the same size after the change.

**Comment by autrilla** [comment]
I added a warning on the README.md in kubernetes/ for what you mentioned.

**Pull request merged**